src/data/bedrock/item/ItemTypeNames.php

```diff
@@ -161,6 +161,7 @@ final class ItemTypeNames{
 	public const DARK_OAK_DOOR = "minecraft:dark_oak_door";
 	public const DARK_OAK_HANGING_SIGN = "minecraft:dark_oak_hanging_sign";
 	public const DARK_OAK_SIGN = "minecraft:dark_oak_sign";
+	public const DEBUG_STICK = "minecraft:debug_stick";
```

Comment:
My local CI wasn't failing, also pretty sure debug_stick is an item that only exists on preview releases

Reply:
Neither was mine, but it seems like it's required for something since the CI on our fork took it very personally